data/frame.go

```diff
@@ -386,8 +388,15 @@ func FrameTestCompareOptions() []cmp.Option {
 		return true
 	})
 
+	metas := cmp.Comparer(func(x, y *FrameMeta) bool {
+		xJSON, _ := json.Marshal(x)
```

Comment:
can you add a comment explaining why this is here.  Speciffically that it is only used for `custom` struct vs map[string]

Reply:
just added a comment 👍